Submitted By: Arne Fitzenreiter <arne_f@ipfire.org>
Date: 2008-08-17
Description: Add devolo MicroLink ADSL Fun USB id's to eagle-atm driver 

diff -Naur linux-2.6.23.17.org/drivers/usb/atm/ueagle-atm.c linux-2.6.23.17/drivers/usb/atm/ueagle-atm.c
--- linux-2.6.23.17.org/drivers/usb/atm/ueagle-atm.c	2008-02-26 01:14:28.000000000 +0100
+++ linux-2.6.23.17/drivers/usb/atm/ueagle-atm.c	2008-08-17 12:35:03.000000000 +0200
@@ -174,6 +174,16 @@
 #define ELSA_PID_PSTFIRM	0x3350
 #define ELSA_PID_PREFIRM	0x3351
 
+#define DEVOLO_VID		0x1039
+#define DEVOLO_E1I_PID_PSTFIRM	0x2100
+#define DEVOLO_E1I_PID_PREFIRM	0x2101
+#define DEVOLO_E1P_PID_PSTFIRM	0x2110
+#define DEVOLO_E1P_PID_PREFIRM	0x2111
+#define DEVOLO_E2I_PID_PSTFIRM	0x2120
+#define DEVOLO_E2I_PID_PREFIRM	0x2121
+#define DEVOLO_E2P_PID_PSTFIRM	0x2130
+#define DEVOLO_E2P_PID_PREFIRM	0x2131
+
 /*
  * Sagem USB IDs
  */
@@ -1795,6 +1805,14 @@
 static const struct usb_device_id uea_ids[] = {
 	{USB_DEVICE(ELSA_VID,	ELSA_PID_PREFIRM),	.driver_info = ADI930 | PREFIRM},
 	{USB_DEVICE(ELSA_VID,	ELSA_PID_PSTFIRM),	.driver_info = ADI930 | PSTFIRM},
+	{USB_DEVICE(DEVOLO_VID,	DEVOLO_E1I_PID_PREFIRM),.driver_info = EAGLE_I | PREFIRM},
+	{USB_DEVICE(DEVOLO_VID,	DEVOLO_E1I_PID_PSTFIRM),.driver_info = EAGLE_I | PSTFIRM},
+	{USB_DEVICE(DEVOLO_VID,	DEVOLO_E1P_PID_PREFIRM),.driver_info = EAGLE_I | PREFIRM},
+	{USB_DEVICE(DEVOLO_VID,	DEVOLO_E1P_PID_PSTFIRM),.driver_info = EAGLE_I | PSTFIRM},
+	{USB_DEVICE(DEVOLO_VID,	DEVOLO_E2I_PID_PREFIRM),.driver_info = EAGLE_II | PREFIRM},
+	{USB_DEVICE(DEVOLO_VID,	DEVOLO_E2I_PID_PSTFIRM),.driver_info = EAGLE_II | PSTFIRM},
+	{USB_DEVICE(DEVOLO_VID,	DEVOLO_E2P_PID_PREFIRM),.driver_info = EAGLE_II | PREFIRM},
+	{USB_DEVICE(DEVOLO_VID,	DEVOLO_E2P_PID_PSTFIRM),.driver_info = EAGLE_II | PSTFIRM},
 	{USB_DEVICE(EAGLE_VID,	EAGLE_I_PID_PREFIRM),	.driver_info = EAGLE_I | PREFIRM},
 	{USB_DEVICE(EAGLE_VID,	EAGLE_I_PID_PSTFIRM),	.driver_info = EAGLE_I | PSTFIRM},
 	{USB_DEVICE(EAGLE_VID,	EAGLE_II_PID_PREFIRM),	.driver_info = EAGLE_II | PREFIRM},
